lib/galaxy/model/migrate/versions/0142_add_columns_for_deleted_disk_usage.py

```diff
@@ -0,0 +1,37 @@
+"""
+Migration script to and deleted_disk_usage' column to the User and GalaxySession
```

Comment:
s/to and deleted_disk_usage'/to add deleted_disk_usage

Reply:
Good catch. I'll fix it.